### PR TITLE
Move from Winston to Loglevel

### PR DESCRIPTION
**Pull request by adhusson**
Currently replaced Winston with loglevel. If fine, will 

- [ ] publish the commonlib package and 
- [ ] remove mangrove.js' logger.ts

### CODE REVIEW

File: packages/mangrove-solidity/config/default.js [hunk 1]
```diff
@@ -53,6 +53,11 @@ config.hardhat = {
         enabled: true,
         runs: 20000,
       },
+      outputSelection: {
+        "*": {
+          "*": ["storageLayout"],
+        },
+      },
```
Comment (espendk) [inline packages/mangrove-solidity/config/default.js:60]: This seems unrelated to the rest of the PR?

File: packages/commonlib.js/package.json [hunk 2]
```diff
@@ -25,11 +25,13 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "dependencies": {
```
Comment (espendk) [inline packages/commonlib.js/package.json:27]: I'd expect to see `loglevel` added here as it's now a direct dependency.

File: packages/commonlib.js/src/logger.ts [hunk 3]
```diff
@@ -1,65 +1,109 @@
...
-  logLevel: string,
-  additionalTransports: Transport[] = []
-): BetterLogger => {
-  const consoleTransport = new transports.Console({
-    level: logLevel,
-    handleExceptions: true,
-    format: format.combine(
-      format.colorize(),
+  logLevel: string
+  //additionalTransports: Transport[] = []
```
Comment (espendk) [inline packages/commonlib.js/src/logger.ts:43]: Since we're deleting Winston, I think we should remove this - it's no longer relevant.

**Review by espendk — CHANGES_REQUESTED** [review]
Looks good. Have added a few minor suggestions.

File: packages/mangrove-solidity/config/default.js [hunk 4]
```diff
@@ -53,6 +53,11 @@ config.hardhat = {
         enabled: true,
         runs: 20000,
       },
+      outputSelection: {
```
Comment (dontrolle) [inline packages/mangrove-solidity/config/default.js:56]: Apparently my comment from earlier today got deleted, so I'll repost it:
> What does this do?

**Review by dontrolle — COMMENTED** [review]
Apart from the minor comments, looks good.

File: packages/commonlib.js/src/logger.ts (hunk 3 above)
Comment (adhusson) [reply packages/commonlib.js/src/logger.ts:43]: Do you mean removing the commented out `additionalTransports...` ?

File: packages/mangrove-solidity/config/default.js (hunk 4 above)
Comment (adhusson) [reply packages/mangrove-solidity/config/default.js:56]: This should not be here :)

File: packages/commonlib.js/src/logger.ts (hunk 3 above)
Comment (espendk) [reply packages/commonlib.js/src/logger.ts:43]: Yes, that's what I mean, sorry for being imprecise :-) (in my defense, GitHub is retarded - I've commented on only that line, but it's making it look like I've commented on a two-line diff 🙄)

**Comment by adhusson** [comment]
I have added a fix to properly setup the loglevel, and make mangrove.js use the published version on commonlib.

I think next steps are you give a go to the bots on this branch and if everything is fine we can merge.

**Review by espendk — COMMENTED** [review]
I've renamed commonlib-js to commonlib.js.

I'll test logging in a bot and then approve.

**Review by espendk — APPROVED** [review]
I just checked the cleaning bot and the logs look ok.
We still need to verify once this has been deployed to Heroku.

**Pull request merged**